code/game/jobs/job/outsider/representative.dm

```diff
@@ -142,3 +143,11 @@
 	if(citizenship)
 		rep_objectives = citizenship.get_objectives(mission_level, H)
 	return rep_objectives
+
+/datum/job/consular/after_spawn(mob/living/carbon/human/H)
+	var/datum/citizenship/citizenship = SSrecords.citizenships[H.citizenship]
+	LAZYADD(blacklisted_citizenship, citizenship.name)
```

Comment:
How does this behave when both consulars are ready at roundstart? Is it handled correctly that one is sent back to the lobby and the other slot isn't taken?

Reply:
Currently, it would not be handled correctly if spawn_positions was set to 2. I will try and find a way to fix this before the actual job slot increase is opened, but if I can't then leaving spawn_positions as 1 for these jobs should act as a bandaid.

Reply:
this has now been addressed